homeassistant/components/xiaomi_ble/sensor.py

```diff
@@ -68,6 +76,20 @@
         state_class=SensorStateClass.MEASUREMENT,
         entity_registry_enabled_default=False,
     ),
+    # Used for e.g. moisture sensor on HHCCJCY01
```

Comment:
Candidate for a new sensor device class? I count 7 existing integrations that have moisture sensors.

Reply:
Good idea. I'll add it to my todo.